### PR TITLE
Use formatted float in `dataframe_to_wiki`

### PR DESCRIPTION
**Pull request by dachengx**
## What does the code in this PR do / what does it improve?

In the previous implementation, `straxen.dataframe_to_wiki` will give you '0.5' when `float_digits=2`, which gives the output different lengths. 

Previous:
```
import pandas as pd
import straxen

df = pd.DataFrame({'x': [0.51], 'y': [0.50]})
print(straxen.dataframe_to_wiki(df, float_digits=2))
```

gives output:
```
^ Awesome table ^^
^ x ^ y ^
| 0.51 | 0.5 |
```

## Can you give a minimal working example (or illustrate with a figure)?

Now the new code gives output:
```
^ Awesome table ^^
^ x ^ y ^
| 0.51 | 0.50 |
```

_Please include the following if applicable:_
  - [ ] _Update the docstring(s)_
  - [ ] _Update the documentation_
  - [ ] _Tests to check the (new) code is working as desired._
  - [ ] _Does it solve one of the open issues on github?_

### _Notes on testing_
 - _Until the automated tests pass, please mark the PR as a draft._
 - _On the XENONnT fork we test with database access, on private forks there is no database access for security considerations._

All _italic_ comments can be removed from this template.

### CODE REVIEW

**Comment by JYangQi00** [comment]
Hey Dacheng, I read through the commits and they look fine to me. Correct me if I'm missing something but, it seems like the failing tests are unrelated to your commits?

File: straxen/misc.py [hunk 1]
```diff
@@ -31,27 +31,28 @@
 
 
 @export
-def dataframe_to_wiki(df, float_digits=5, title='Awesome table', 
-                      force_int=tuple()):
+def dataframe_to_wiki(
+    df, float_digits=5, title='Awesome table', force_int=()):
```
Comment (WenzDaniel) [inline straxen/misc.py:35]: General question, should we do explicit typing whenever we change one of those functions? I guess you know what I mean, just for reference: https://docs.python.org/3/library/typing.html

File: straxen/misc.py [hunk 2]
```diff
@@ -31,27 +31,28 @@
...
-    def do_round(x):
+    def format_float(x):
         if isinstance(x, float):
-            return round(x, float_digits)
+            return f'{x:.{float_digits}f}'
         return x
-    force_int = np.where(np.in1d(df.columns.values,
-                                 strax.to_str_tuple(force_int)))[0]
+    force_int = np.where(np.in1d(
+        df.columns.values, strax.to_str_tuple(force_int)))[0]
```
Comment (WenzDaniel) [inline straxen/misc.py:51]: General comment for future code, no need to change here: this guy is in general a bit ugly many operations for a single line.

**Review by WenzDaniel — APPROVED** [review]
Looks good.

**Comment by WenzDaniel** [comment]
The error seems to be related to the holoviews displays, but I do not know why.

File: straxen/misc.py (hunk 2 above)
Comment (JYangQi00) [reply straxen/misc.py:51]: Just wondering, numpy's documentation says that np.isin is preferable to np.in1d, but does it have any functional differences that we should worry about?

**Review by JYangQi00 — APPROVED** [review]
Everything seems to work for me! 👍

File: straxen/misc.py (hunk 2 above)
Comment (dachengx) [reply straxen/misc.py:51]: `in1d` is much faster as they said https://numpy.org/doc/stable/reference/generated/numpy.isin.html#numpy.isin

Comment (JYangQi00) [reply straxen/misc.py:51]: Gotcha, thanks!

File: straxen/misc.py (hunk 1 above)
Comment (dachengx) [reply straxen/misc.py]: Updated.

**Pull request merged**